### PR TITLE
Add search to the index talks page

### PR DESCRIPTION
**Pull request by Sayok**
### Description

there is no search bar on the pivorak.com/talks page, it should be added to have navigation through talks faster.

a search bar has been added to the pivorak.com/talks page, test steps and changes are described below.

### How to test instructions

1. go to /talks page
2. verify that search bar shown up on the page (see attachments)
3. put some title of the talks and press search button
4. talks with appropriated title or description should be shown up on the page.

### Pre-merge checklist
 
- [ improvement] The PR relates to a single subject with a clear title and description
- [ +] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

| Before                                      | After                                       |
| ------------------------------------------- | ------------------------------------------- |
| 
before_talks | after_search
 |

### CODE REVIEW

File: spec/features/talks/search_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+RSpec.describe 'Talks search' do
+  let(:event) { create :event, title: 'Event Title' }
+
+  let!(:talk_1) { create :talk, title: 'Talk 1', description: "infrastructure", event: event }
```
Comment (sveredyuk) [inline spec/features/talks/search_spec.rb]: `let!(:talk_1) { create :talk, title: title_of_talk_1', description: "infrastructure", event: event }`
and cetera

**Review by denys-medynskyi — CHANGES_REQUESTED** [review]
Good from the code perspective.
However, you forgot to add text when no results are present. Please add feature spec for this.
How to reproduce:
Visit - http://localhost:3000/talks?utf8=%E2%9C%93&query=asdada
Check this screenshot : https://cl.ly/3t42051N2r0a

**Comment by Sayok** [comment]
{quote } @denysmedynskyi However, you forgot to add text when no results are present. Please add feature spec for this. {quote}

will fix it soon.

**Review by denys-medynskyi — APPROVED** [review]
Nice! Thank you.

**Comment by denys-medynskyi** [comment]
I am going to merge it after CI is green

**Pull request merged**